src/importexport/capella/internal/capella.cpp

```diff
@@ -290,6 +290,7 @@ static void processBasicDrawObj(QList<BasicDrawObj*> objects, Segment* s, int tr
             text->setFamily(f.family());
             text->setItalic(f.italic());
             // text->setUnderline(f.underline());
```

Comment:
Not sure why this is commented out (it isn't in 3.x), but because of that I left the below line commentd out too